lib/reducer.ts

```diff
@@ -70,23 +74,36 @@ export const propertiesIntrospectionFieldReducer:
 export const definitionsIntrospectionFieldReducer:
     MemoListIterator<IntrospectionFieldReducerItem, JSONSchema6Acc, ReadonlyArray<IntrospectionFieldReducerItem>> =
     (acc, curr: IntrospectionFieldReducerItem): JSONSchema6Acc => {
-
         if (isIntrospectionField(curr)) {
-
             const returnType = isNonNullIntrospectionType(curr.type) ?
                 graphqlToJSONType(curr.type.ofType) :
                 graphqlToJSONType(curr.type);
+
             acc[curr.name] = returnType;
         } else if (isIntrospectionInputValue(curr)) {
             const returnType = isNonNullIntrospectionType(curr.type) ?
                 graphqlToJSONType(curr.type.ofType) :
                 graphqlToJSONType(curr.type);
+
             acc[curr.name] = returnType;
+            if (curr.defaultValue) {
+                acc[curr.name].default = resolveDefaultValue(curr)
+            }
         }
+
         acc[curr.name].description = curr.description || undefined;
         return acc;
     };
 
+// ENUM type defaults will not JSON.parse correctly, so if it is an ENUM then don't
+// try to do that.
+// TODO: fix typing here
+export const resolveDefaultValue = (curr) => {
+    return isIntrospectionEnumType(curr.type) ?
+        curr.defaultValue :
```

Comment:
This is how I chose to do this part. Could make more sense to do something else? I could see as a consumer wanting/needing to know that this type was an `ENUM` so that I'd know what to do with it (e.g. don't handle it as a string even though the value is a string) without resolving the `$ref` to find out it was an `ENUM`.

Couldn't think of anything better that was easy enough for now.